templates/home.html

```diff
@@ -221,17 +250,22 @@ <h5 class="modal-title" id="title"></h5>
 
         <template id="requestTemplate">
             <li class="list-group-item request" id="">
-                <div class="d-flex w-100 justify-content-between">
+                <div class="d-flex w-100 justify-content-between" id="requestHeader">
                     <div>
                         <h5 class="mb-1" id="title"></h5>
                     </div>
                     <small class="text-nowrap" id="smallText" style="text-align: right;"></small>
                 </div>
                 <p style="margin-bottom: 0.2em" id="description"></p>
-                <div class="d-flex justify-content-between flex-wrap">
+                <div class="d-flex justify-content-between flex-wrap" id="requestFooter">
                     <small style="margin-bottom: 0.2em" id="author"></small>
                     <div class="d-flex flex-nowrap ml-auto">
-                        <span class="ml-2">
+                        {% if admin %}
+                        <span class="ml-2" id="mergeSpan">
+                            <button type="button" class="btn btn-warning" onclick="addRequestToList(this)" id="selectToMergeButton" requestId="">Select to merge</button>
+                        </span>
+                        {% endif %}
+                        <span class="ml-2" id="detailsSpan">

```

Comment:
having this button always visible is a bit excessive, I think there should be a button in the details popup to "start a merge" which adds the request to the merge list and then only show this button if there is at least one item in the merge list.

additionally, its a bit confusing that the button is still available after the request is selected. there should be some indicator that tells the user the request is selected, probably changing the button into a deselect button and maybe changing some color to make it extra clear?

Reply:
i added a checkbox on toolbar to show/hide all buttons related to merging